pkg/controllers/flannelmigration/network_migrator.go

```diff
@@ -233,6 +233,15 @@ func (m *networkMigrator) MigrateNodes(nodes []*v1.Node) error {
 
 		err = m.setupCalicoNetworkForNode(node)
 		if err != nil {
+			// Reset node label to `network=none` so that migration controller will try to
```

Comment:
What happens if the controller dies before this line?  Will the restarted controller skip to the next node?

Reply:
Yes. Good point.

Seems using just one label `network-during-migration=calico/flannel/none` for both daemonset nodeSelector and process control is not possible. I will add another label to mark that migration is not complete for node.  So the whole process would like that.
- Start to migrate one node.
- Set node label `migration-in-process=true`.
- Set node label `network-during-migration=none` to evict Flannel daemonset pod.
- Remove flannel network, drain node etc.
- Set node label `network-during-migration=calico` to allow calico-node pod running on the node.
- Check calico-node pod ready and `vxlan.calico` device created.
- Remove node label `migration-in-process`.
- Move on to next node.

Then the main thread will check if any node with node label `migration-in-process` and process that node after restart.